.github/workflows/wheels.yaml

```diff
@@ -33,6 +33,21 @@ jobs:
           name: wheels
           path: wheelhouse
 
+  upload_all:
+    needs: [build_wheels]
+    environment: pypi
+    permissions:
+      id-token: write
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'published'
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: artifact
```

Comment:
This should probably say `name: wheels` to match the artifact name on L33.

Reply:
Good catch, thanks!